libs/ledger-state/bench/Performance.hs

```diff
@@ -129,8 +129,8 @@ main = do
     , env (pure utxo) $ \utxo' ->
         bgroup
           "UTxO"
-          [ bench "balance" $ nf balance utxo'
-          , bench "coinBalance" $ nf coinBalance utxo'
+          [ bench "balance" $ nf sumUTxO utxo'
+          , bench "coinBalance" $ nf sumCoinUTxO utxo'
```

Comment:
Maybe you want to change the name of the benches as well, to make it easier to correlate with the code? 

```suggestion
          [ bench "sumUTxO" $ nf sumUTxO utxo'
          , bench "sumCoinUTxO" $ nf sumCoinUTxO utxo'
```